src/net/rpc.rs

```diff
@@ -3397,6 +3498,175 @@ mod test {
+                let peer_server_sortdb = peer_server.sortdb.take().unwrap();
+
+                if peer_server.network.inv_state.is_none() {
+                    peer_server.network.init_inv_sync(&peer_server_sortdb);
+                }
+                let peer_client_nk = peer_client.to_neighbor().addr;
+
+                if let Some(ref mut inv_state) = peer_server.network.inv_state {
+                    if inv_state.get_stats(&peer_client_nk).is_none() {
+                        inv_state.add_peer(peer_client_nk.clone(), true);
+                    }
+                    inv_state
+                        .get_stats_mut(&peer_client_nk)
+                        .unwrap()
+                        .inv
+                        .num_sortitions = 1;
+                }
+                peer_server.sortdb = Some(peer_server_sortdb);
+                convo_client.new_get_health()
+            },
+            |ref http_request, ref http_response, ref mut peer_client, ref mut peer_server| {
+                let req_md = http_request.metadata().clone();
+                match http_response {
+                    HttpResponseType::GetHealth(response_md, data) => {
+                        assert_eq!(data.is_healthy, true);
+                        assert_eq!(data.percent_of_blocks_synced, 100);
+                        true
+                    }
+                    _ => {
+                        error!("Invalid response: {:?}", &http_response);
+                        false
+                    }
+                }
+            },
+        );
+    }
+
+    #[test]
+    #[ignore]
+    fn test_rpc_get_health_no_stats() {
+        test_rpc(
+            "test_rpc_get_health",
+            40010,
+            40011,
+            50010,
+            50011,
+            |ref mut peer_client,
+             ref mut convo_client,
+             ref mut peer_server,
+             ref mut convo_server| { convo_client.new_get_health() },
+            |ref http_request, ref http_response, ref mut peer_client, ref mut peer_server| {
+                let req_md = http_request.metadata().clone();
+                match http_response {
+                    HttpResponseType::NotFound(response_md, msg) => {
+                        assert_eq!(msg, "Peer block stats not found.");
+                        true
+                    }
+                    _ => {
+                        error!("Invalid response: {:?}", &http_response);
+                        false
+                    }
+                }
+            },
+        );
+    }
+
+    #[test]
+    #[ignore]
+    fn test_rpc_get_health_no_stats_for_initial_peers() {
+        test_rpc(
+            "test_rpc_get_health",
+            40010,
+            40011,
+            50010,
+            50011,
+            |ref mut peer_client,
+             ref mut convo_client,
+             ref mut peer_server,
+             ref mut convo_server| {
+                // // create block stats for peer server
```

Comment:
Is this comment really more central than for any other line? Is it copied from somewhere? Why is it double quotes?

Reply:
removed the double quotes, also not copied from anywhere. I added it to give context to the setup code.